minimal/docker-compose.yml

```diff
@@ -41,21 +39,24 @@ services:
     build: ./compose/nginx/production
     depends_on:
       - {{ project_name }}_web
-    volumes:
-      - ./media:/code/media
-      - ./staticfiles:/code/staticfiles
+    volumes_from:
+      - {{ project_name }}_web
+      - {{ project_name }}_certbot
     ports:
       - "0.0.0.0:80:80"
 
   {{ project_name }}_certbot:
     restart: always
     build: ./compose/certbot/production
-    depends_on:
-      - {{ project_name }}_nginx
+    volumes:
+      - {{ project_name }}_production_certs:/etc/letsencrypt
     volumes_from:
+      - {{ project_name }}_web
+    depends_on:
       - {{ project_name }}_nginx
+      - {{ project_name }}_web
     env_file: production.env
 
-
 volumes:
   {{ project_name }}_data:
+  {{ project_name }}_production_certs:
```

Comment:
@pslobo can't we just have a `{{ project_name }}_certs` ? your not supposed to run staging and prod on the same machine so i think `{{ project_name }}_certs`  should be ok